src/ImageService.Validation/Helpers/EncodeHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ImageService.Validation.Helpers
+{
+  public static class EncodeHelper
```

Comment:
i do not sure what create a static class is good idea

Reply:
done